Assets/Character/AaronTestAssets/PlayerState.cs

```diff
@@ -0,0 +1,74 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class PlayerState : MonoBehaviour {
+
+
+    private Vector3 scale;
+    protected float direction = 1;
+    protected int facing = 1;
```

Comment:
Actually, you're right that they all seem to have their own idea of direction which isn't really intended. Does making this public fix that? I was having a hard time getting the direction to be correct across all states and this is probably why.